boards/samr21-xpro/dist/gdb.conf

```diff
@@ -0,0 +1,3 @@
+target remote localhost:3333
+monitor reset halt
+load
```

Comment:
To be honest, I'm not entirely sure what load does exactly. I was under the impression that it loads debug symbols from the device or something akin. Is that correct?